akka-http-core/src/main/scala/akka/http/scaladsl/model/ws/Message.scala

```diff
@@ -13,9 +13,6 @@ import akka.util.ByteString
  */
 sealed trait Message
 
-/**
- * A binary
- */
 sealed trait TextMessage extends Message {
```

Comment:
Please add a ticket to add proper docs for those types; they're pretty core to WS handling so should have docs (even if boring)